Project/src/main/resources/templates/reviewer.html

```diff
@@ -4,25 +4,68 @@
     <meta charset="UTF-8">
     <title>Reviewer</title>
     <link rel="stylesheet" href="webjars/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="webjars/datatables/css/dataTables.bootstrap4.min.css">
+    <link rel="stylesheet" href="/css/upload.css"/>
+    <script src="webjars/jquery/jquery.min.js"></script>
+    <script src="webjars/bootstrap/js/bootstrap.bundle.min.js"></script>
+    <script src="/js/reviewModal.js"></script>
+    <script src="/js/upload.js"></script>
 </head>
 <body>
     <header th:insert="header.html :: header"></header>
-    <table>
+    <table class="table table-bordered" id="reviewer_table">
         <thead>
-        <th>ID</th>
         <th>Filename</th>
         <th>Status</th>
         <th>Due Date</th>
+        <th>Submission Date</th>
         </thead>
         <tbody>
         <tr th:each="article: ${articles}">
-            <td th:text="${article.id}"></td>
             <td th:text="${article.fileName}"></td>
             <td th:text="${article.state}"></td>
+            <td th:text="${article.reviewDueDate}"></td>
             <td th:text="${article.date}"></td>
-            <td><a th:href="@{/review/{id}(id=${article.id})}">Submit Review</a></td>
+            <td th:switch="${article.review}">
+                <button th:case="NULL" class="btn btn-primary" data-toggle="modal" data-target="#reviewModal" th:data-id="${article.id}">Submit Review</button>
+                <div th:case="*">
+                    <a class="btn btn-primary" th:href="@{/review/{id}(id=${article.id})}">Download Review</a>
+                    <button class="btn btn-primary" data-toggle="modal" data-target="#reviewModal" th:data-id="${article.id}">Edit Submission</button>
```

Comment:
![image](https://user-images.githubusercontent.com/10526581/54641357-6569d680-4a68-11e9-9bf2-399072d58c11.png)

The CSS after you submit a file is reaaally ugly. Can you fix that spacing?

Reply:
Just do this last change and then merge, I accepted these changes on the basis that you will change above